scripts/buildtools/versions

```diff
@@ -4,3 +4,4 @@ github.com/algorand/msgp v1.1.52
 github.com/algorand/oapi-codegen v1.3.7
 github.com/go-swagger/go-swagger v0.25.0
 gotest.tools/gotestsum v1.6.4
+github.com/golangci/golangci-lint/cmd/golangci-lint v1.41.1
```

Comment:
would you like to take this opportunity to upgrade to 1.48.0?

Reply:
We'll need a new policy for installing developer dependencies for that. We would need to install the binary release directly instead of building because it's using go 1.19.

Reply:
ohh OK, nevermind